vkHW/vkHW/Views/News/NewsTableViewController.swift

```diff
@@ -53,37 +39,86 @@ final class NewsTableViewController: UITableViewController {
     // MARK: - Public Methods
 
     override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        posts.count
+        3
```

Comment:
```suggestion
        3
```
Можно посчитать как "магическое число" Лучше возьми количество типов ячеек.